pkgs/development/tools/misc/luarocks/luarocks-nix.nix

```diff
@@ -16,7 +16,7 @@ luarocks_bootstrap.overrideAttrs (old: {
     hash = "sha256-hsjv+jlLsoIDM4gB/0mFeoVu1YZ1I9ELDALLTEnlCF0=";
   };
 
-  propagatedBuildInputs = old.propagatedBuildInputs ++ [
+  propagatedNativeBuildInputs = old.propagatedNativeBuildInputs ++ [
```

Comment:
question: are you sure `file` and `nurl` should be `propagatedNativeBuildInputs` too?

Reply:
they should just be in the wrapper. proabaly for me to fix though.